### PR TITLE
Allow get_tables to see views and v10-style partitioned tables

### PR DESCRIPTION
**Pull request by coreyhuinker**
And it does so with just one query.

### CODE REVIEW

**Comment by coreyhuinker** [comment]
tl;dr - yes

The catalog tables (pg_class, pg_attribute, pg_namespace) are present in all supported version of postgresql (and have probably been there since postgres started).

There have been no instances of postgres changing a `relkind` value from version to version, and it would be a major upgrade headache if they did try doing that.

While partitions don't exist prior to v10, there's no harm in running a query that would not filter out the nonexistent rows.

We may have to tweak the `IN` clause when new discoverable object types are made available in future version of postgres, but that's about it.

That, and the existing code risked duplicate entries for mviews if a future version of information_schema decided that mviews were tables.

This query should also work for the postgresql forks (Vertica, Redshift) though they may have chosen different relkind values for the object types.

**Comment by arikfr** [comment]
Thank you for the pull request and the detailed explanation!
